### ISSUE TITLE
Ability to set priority of the assemblers (1.7.10)

### ISSUE THREAD

**Issue by DoomSquirter** [Mordenkainen/EquivalentEnergistics#79]
watched your vid and noticed you can get the assemblers to take precedence if you set your interface/crafters to a lower priority than the ee assemblers.  would be nice to be able to adjust the actual ee assembler instead.  I have a very well entrenched setup and going to each of them to set priorities is going to be a nightmare.

**Comment by Mordenkainen** [comment]
One of the design goals was no UI for this stuff. So setting priority is problematic.

**Comment by DoomSquirter** [comment]
yeah, I figured as much.  Possibly adding an item you can hit them with maybe and set the priority within the item so you don't have to make guis for the blocks?  I basically went around and changed all my interfaces, so even if you can't do that, at least it's a one-off kind of thing, and if you're aware of this while you're building it you can prepare before you bury it in blocks like I did.  

Thanks alot, feel free to close.
